docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -527,6 +529,25 @@ The following things are validated:
 
 Any errors found will be logged with error level to ``stderr``.
 
+scrub
+^^^^^
+
+Rewrites the sstable, skipping or fixing corrupt parts. Not all kind of corruptions can be skipped or fixed by scrub.
+Currently this is limited to ordering issues on the partition, row or mutation-fragment level, see `sstable content <scylla-sstable-sstable-content_>`_ for more details.
+
+Scrub has several modes:
+
+* **abort** - abort the scrub as soon as any error is found (recognized or not). This mode is not all that useful (**validate** mode is strictly an improvement over **abort** mode), it is included for completeness' sake.
+* **skip** - skip over any corruptions found, thus omitting them from the output. Note that skip can end up omitting more than was strictly necessary, but it is guaranteed that it will skip all detectable corruptions.
+* **segregate** - fix partition/row/mutation-fragment out-of-order errors, by segregating the output into as many sstables as required, such that the content of each output sstable is properly ordered.
```

Comment:
```suggestion
* **segregate** - Fixes partition/row/mutation-fragment out-of-order errors by segregating the output into as many SStables as required so that the content of each output SStable is properly ordered.
```